ztp/source-crs/extra-manifest/image-registry-partition-mc.yaml.tmpl

```diff
@@ -0,0 +1,52 @@
+{{- range .Data.Nodes -}}
+{{ if .DiskPartition -}}
+{{- "# Automatically generated by extra-manifests-builder" }}
+{{ "# Do not make changes directly." }}
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 98-var-imageregistry-partition-{{ $.Role }}
```

Comment:
If there are multiple nodes does each MachineConfig need a unique name per node?

Reply:
Future-proof is always prudent +1